akka-remote-tests/src/multi-jvm/scala/akka/remote/artery/MaxThroughputSpec.scala

```diff
@@ -55,6 +54,12 @@ object MaxThroughputSpec extends MultiNodeConfig {
            # See akka-remote-tests/src/test/resources/aeron.properties
            #advanced.embedded-media-driver = off
            #advanced.aeron-dir = "target/aeron"
+           
+           #advanced.compression {
+           #  enabled = on
+           #  actor-refs.enabled = on
+           #  manifests.enabled = on
+           #}
          }
        }
```

Comment:
Advertised table is:

```
[InboundCompressionTable(akka://MaxThroughputSpec)] Advertise ActorRef compression [CompressionTable(1,
Map(Actor[artery://MaxThroughputSpec@localhost:58601/user/5-to-5-snd4#-798311822] -> 9, 
Actor[akka://MaxThroughputSpec/user/5-to-5-rcv4#133185920] -> 8, 
Actor[artery://MaxThroughputSpec@localhost:58601/system/testActor-1#7166865] -> 1, 
Actor[akka://MaxThroughputSpec/user/5-to-5-rcv1#-88748461] -> 2, 
Actor[akka://MaxThroughputSpec/user/5-to-5-rcv5#1348321425] -> 10, 
Actor[akka://MaxThroughputSpec/user/5-to-5-rcv2#-1319771620] -> 4, 
Actor[artery://MaxThroughputSpec@localhost:58601/user/5-to-5-snd3#-1135029393] -> 7, 
Actor[akka://MaxThroughputSpec/user/5-to-5-rcv3#865004274] -> 6, 
Actor[artery://MaxThroughputSpec@localhost:58601/user/5-to-5-snd5#499982012] -> 11, 
Actor[artery://MaxThroughputSpec@localhost:58601/user/5-to-5-snd2#647171527] -> 5, 
Actor[artery://MaxThroughputSpec@localhost:58601/user/5-to-5-snd1#1751833096] -> 3, 
Actor[akka://MaxThroughputSpec/deadLetters] -> 0
))] to [artery://MaxThroughputSpec@localhost:58601]
```

It does slow down currently, so I'll work on that next (addressing many of the TODOs etc):

with: `[JVM-1] === MaxThroughput 5-to-5-snd5: throughput 96,177 msg/s, 9,617,697 bytes/s, dropped 0, max round-trip 116 ms, burst size 200, payload size 100, 20795 ms to deliver 400000 messages`

without: `[JVM-1] === MaxThroughput 5-to-5-snd5: throughput 171,821 msg/s, 17,182,131 bytes/s, dropped 0, max round-trip 109 ms, burst size 200, payload size 100, 11640 ms to deliver 400000 messages`

I'll work on profiling now